tests/v1/attention/test_b12x_sparse_mla_api.py

```
@@ -197,6 +197,93 @@ def test_b12x_glm5_next_keeps_hybrid_manager_page_unsplit() -> None:
+        cache_config=SimpleNamespace(
+            block_size=256,
+            mamba_block_size=None,
+            mamba_cache_mode="align",
+            mamba_page_size_padded=1234,
+            prefix_cache_retention_interval=4096,
+        ),
+    )
+    monkeypatch.setenv("VLLM_GLM53_SPLIT_TARGET_BLOCK_SIZE", "auto")
+    monkeypatch.setenv("VLLM_GLM53_SPLIT_MAMBA_BLOCK_SIZE", "auto")
+
+    Platform._align_hybrid_block_size(config, B12xGLM5NextMLASparseBackend)
```

Ruff (0.16.3)

[error] Undefined name `Platform`

(F821)